src/test/unit/mixer_tricopter_unittest.cc

```diff
@@ -21,55 +21,162 @@
+        servoConf.max = DEFAULT_SERVO_MAX;
+        servoConf.middle = DEFAULT_SERVO_MIDDLE;
+        servoConf.rate = 100;
+        servoConf.forwardFromChannel = CHANNEL_FORWARDING_DISABLED;
+
+        // give all servos a default command
+        for (uint8_t i = 0; i < MAX_SUPPORTED_SERVOS; i++) {
+            servo[i] = DEFAULT_SERVO_MIDDLE;
+        }
+
+        triInitMixer(&servoConf, &servo[5], &mixerConfig);
+        tailTune.mode = TT_MODE_THRUST_TORQUE;
+        tailTune.tt.state = TT_WAIT_FOR_DISARM;
+        tailTune.tt.servoAvgAngle.numOf = 300;
+    }
+};
 
-TEST(ATest, test) {
-    ASSERT_TRUE(false);
+TEST_F(BasicTest, TestThrustFactorCalculation139) {
+    // given
+    // expecting tri_tail_motor_thrustfactor = 1 / tan(angle)
+    tailTune.tt.servoAvgAngle.sum = 12345;
+    // and
+    tailTuneModeThrustTorque(&tailTune.tt, true);
+    // then
+    EXPECT_NEAR(139, mixerConfig.tri_tail_motor_thrustfactor, 1);
 }
 
+TEST_F(BasicTest, TestThrustFactorCalculation145) {
+    // given
+    tailTune.tt.servoAvgAngle.sum = 11836;
+    // and
+    tailTuneModeThrustTorque(&tailTune.tt, true);
+    // then
+    EXPECT_NEAR(145, mixerConfig.tri_tail_motor_thrustfactor, 1);
+}
+
+TEST_F(BasicTest, TestThrustFactorCalculation125) {
+    // given
+    tailTune.tt.servoAvgAngle.sum = 13722;
+    // and
+    tailTuneModeThrustTorque(&tailTune.tt, true);
+    // then
+    EXPECT_NEAR(125, mixerConfig.tri_tail_motor_thrustfactor, 1);
+}
+
+TEST_F(BasicTest, TestThrustFactorCalculation80) {
+    // given
+    tailTune.tt.servoAvgAngle.sum = 21375;
+    // and
+    tailTuneModeThrustTorque(&tailTune.tt, true);
+    // then
+    EXPECT_NEAR(80, mixerConfig.tri_tail_motor_thrustfactor, 1);
+}
 
 //STUBS
 extern "C" {
+
+typedef struct master_s {
+} master_t;
+
 float dT;
-uint32_t millis(void) { return 0; }
 uint8_t armingFlags;
 int16_t rcCommand[4];
-uint16_t getCurrentMinthrottle(void){ return 0; }
-void beeper(beeperMode_e mode){}
-bool isRcAxisWithinDeadband(int32_t axis) { return true;}
 uint32_t rcModeActivationMask;
 uint16_t flightModeFlags = 0;
-uint16_t disableFlightMode(flightModeFlags_e mask){return 0;}
-void beeperConfirmationBeeps(uint8_t beepCount){}
-uint16_t enableFlightMode(flightModeFlags_e mask){ return 0; }
+int16_t debug[DEBUG16_VALUE_COUNT];
+gyro_t gyro;
+int32_t gyroADC[XYZ_AXIS_COUNT];
+master_t masterConfig;
 
-typedef struct master_s{
-} master_t;
+uint32_t millis(void) {
+    return 0;
+}
 
-master_t masterConfig;
+uint16_t getCurrentMinthrottle(void) {
+    return 0;
+}
 
-throttleStatus_e calculateThrottleStatus(rxConfig_t *rxConfig, uint16_t deadband3d_throttle) { return (throttleStatus_e)0;}
+void beeper(beeperMode_e mode) {
+    (void) mode;
+}
+
+bool isRcAxisWithinDeadband(int32_t axis) {
+    (void) axis;
+    return true;
+}
+
+uint16_t disableFlightMode(flightModeFlags_e mask) {
+    (void) mask;
+    return 0;
+}
+
+void beeperConfirmationBeeps(uint8_t beepCount) {
+    (void) beepCount;
```

Comment:
An alternative would be to remove the variable name. I think this would work too:
uint16_t disableFlightMode(flightModeFlags_e /* mask */) { 
Its a matter of taste.